src/nccl_ofi_rdma.c

```diff
@@ -494,6 +546,15 @@ static inline rdma_req_bounce_data_t *get_bounce_data(nccl_net_ofi_rdma_req_t *r
 	return &req->bounce_data;
 }
 
+/*
+ * @brief	Return write inline struct of write request
+ */
+static inline rdma_req_rma_op_data_t *get_rma_op_data(nccl_net_ofi_rdma_req_t *req,
```

Comment:
`object_verb()` -- so this should be `req_get_rma_op_data()`.

Reply:
I assume you want to to align similar functions in a preceding commit? E.g., we also have `get_bounce_data()` and others.

Reply:
yeah, there are many terrible names in this code.  You should make sure yours is not one of them (but you don't have to fix all the existing terrible names).

Reply:
Renamed it for `get_rma_op_data()` for now.